profiles/CHANGELOG.md

```diff
@@ -0,0 +1,37 @@
+# Changelog
+
+All notable changes to this project will be documented in this file.
+
+The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/)
+
+## [Unreleased]
+
+## 2020-03-17
```

Comment:
After read [Keep a Changelog](https://keepachangelog.com/en/1.0.0/), I think the date should only applied on versions, not every action.

Reply:
> After read [Keep a Changelog](https://keepachangelog.com/en/1.0.0/), I think the date should only applied on versions, not every action.

You are right. The original template does have version https://github.com/olivierlacan/keep-a-changelog/blob/master/CHANGELOG.md
Do you suggest remove all date from Change logs, for current, only keep 'Added', 'Changed' and 'Remove' list instead?

Reply:
After check https://github.com/Project-OSRM/osrm-backend/blob/master/CHANGELOG.md, I think the best practice maybe write down many contents when initialize, then add oneline to describe the change for every PR, also the PR/ISSUE ID better to be record.

Reply:
After discussion, remove "added", "removed" and "deleted". 
Link PR to each change.